packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-legacy-features/controller.js

```diff
@@ -0,0 +1,21 @@
+export default class NutanixLicenceTileLegacyFeaturesCtrl {
+  /* @ngInject */
+  constructor(atInternet) {
+    this.atInternet = atInternet;
+  }
+
+  $onInit() {
+    this.defaultLicenseLength = 12;
```

Comment:
Maybe move 12 in constant file.

Reply:
It's legacy code and only use in that template so I let as is